### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by cap-bots**
:robot: I have created a release *beep* *boop*
---

<details><summary>db-service: 1.7.0</summary>

### Added

* also support lowercase matchespattern function ([#528](https://github.com/cap-js/cds-dbs/issues/528)) ([6ea574e](https://github.com/cap-js/cds-dbs/commit/6ea574ee67ef5e42e4f8ccbe4fe91b46097de129))
* forUpdate and forShareLock ([#148](https://github.com/cap-js/cds-dbs/issues/148)) ([99a1170](https://github.com/cap-js/cds-dbs/commit/99a1170e61de4fd0c505834c25a9c03fc34da85b))
* **hana:** drop prepared statements after end of transaction ([#537](https://github.com/cap-js/cds-dbs/issues/537)) ([b1f864e](https://github.com/cap-js/cds-dbs/commit/b1f864e0a3a0e5efacd803d3709379cab76d61cc))
* **orderby:** allow to disable collations with [@cds](https://github.com/cds).collate: false ([#492](https://github.com/cap-js/cds-dbs/issues/492)) ([820f971](https://github.com/cap-js/cds-dbs/commit/820f971e1ad21fa8f8ca289c1e29b373365df484))

### Fixed

* **cqn2sql:** Smart quoting of columns inside UPSERT rows ([#519](https://github.com/cap-js/cds-dbs/issues/519)) ([78fe10b](https://github.com/cap-js/cds-dbs/commit/78fe10b1df3691614dc77b1d4f82df10a1d641d3))
* deep delete for views without navigation ([#434](https://github.com/cap-js/cds-dbs/issues/434)) ([3ebc9c2](https://github.com/cap-js/cds-dbs/commit/3ebc9c29362045916c9060528729320c0eae4391))
* Getting rid of quirks mode ([#514](https://github.com/cap-js/cds-dbs/issues/514)) ([c9aa6e8](https://github.com/cap-js/cds-dbs/commit/c9aa6e835761ace38447f37cad6a5f39cb0b910c))
* issue with reused select cqns ([#505](https://github.com/cap-js/cds-dbs/issues/505)) ([916d175](https://github.com/cap-js/cds-dbs/commit/916d1756422f0caf02c323052f2addafed39182a))
* joins without columns are rejected ([#535](https://github.com/cap-js/cds-dbs/issues/535)) ([eb9beda](https://github.com/cap-js/cds-dbs/commit/eb9beda728de60081d7afbfcd49305eeb241f3fb))
* **search:** dont search non string aggregations ([#527](https://github.com/cap-js/cds-dbs/issues/527)) ([c87900c](https://github.com/cap-js/cds-dbs/commit/c87900cb157041a6ff76c45192c1d33180840d0f))
* **search:** search on aggregated results in HAVING clause  ([#524](https://github.com/cap-js/cds-dbs/issues/524)) ([61d348e](https://github.com/cap-js/cds-dbs/commit/61d348ebc2528b7f1c6da8c78a7455a438e1b7cf))
</details>

<details><summary>sqlite: 1.6.0</summary>

### Added

* forUpdate and forShareLock ([#148](https://github.com/cap-js/cds-dbs/issues/148)) ([99a1170](https://github.com/cap-js/cds-dbs/commit/99a1170e61de4fd0c505834c25a9c03fc34da85b))
* **hana:** drop prepared statements after end of transaction ([#537](https://github.com/cap-js/cds-dbs/issues/537)) ([b1f864e](https://github.com/cap-js/cds-dbs/commit/b1f864e0a3a0e5efacd803d3709379cab76d61cc))

### Fixed

* **`sqlite`:** use keyword list from compiler ([#526](https://github.com/cap-js/cds-dbs/issues/526)) ([a227c61](https://github.com/cap-js/cds-dbs/commit/a227c61312ba9e7d6a54944c822d5de0bb2d3f3c))
</details>

<details><summary>postgres: 1.6.0</summary>

### Added

* also support lowercase matchespattern function ([#528](https://github.com/cap-js/cds-dbs/issues/528)) ([6ea574e](https://github.com/cap-js/cds-dbs/commit/6ea574ee67ef5e42e4f8ccbe4fe91b46097de129))
* forUpdate and forShareLock ([#148](https://github.com/cap-js/cds-dbs/issues/148)) ([99a1170](https://github.com/cap-js/cds-dbs/commit/99a1170e61de4fd0c505834c25a9c03fc34da85b))

### Changed

* use new cds build API @sap/cds-dk &gt;= 7.5.0 ([#508](https://github.com/cap-js/cds-dbs/issues/508)) ([ef22ebe](https://github.com/cap-js/cds-dbs/commit/ef22ebe68c6a554d4042a0a19bae3b2e1d56cb01))
</details>

<details><summary>hana: 0.1.0</summary>

### Added

* also support lowercase matchespattern function ([#528](https://github.com/cap-js/cds-dbs/issues/528)) ([6ea574e](https://github.com/cap-js/cds-dbs/commit/6ea574ee67ef5e42e4f8ccbe4fe91b46097de129))
* forUpdate and forShareLock ([#148](https://github.com/cap-js/cds-dbs/issues/148)) ([99a1170](https://github.com/cap-js/cds-dbs/commit/99a1170e61de4fd0c505834c25a9c03fc34da85b))
* **hana:** drop prepared statements after end of transaction ([#537](https://github.com/cap-js/cds-dbs/issues/537)) ([b1f864e](https://github.com/cap-js/cds-dbs/commit/b1f864e0a3a0e5efacd803d3709379cab76d61cc))

### Fixed

* **`hana`:** use keyword list from compiler ([#525](https://github.com/cap-js/cds-dbs/issues/525)) ([c6993d9](https://github.com/cap-js/cds-dbs/commit/c6993d9c0e823d403f49f55cfcfa8663971293c8))
* **hana:** improve search inside where clause detection ([#538](https://github.com/cap-js/cds-dbs/issues/538)) ([51b8af3](https://github.com/cap-js/cds-dbs/commit/51b8af3b42fba3f7b509d00675997d6c711cacc4))
* **hana:** reduce service manager calls for failing tenants ([#533](https://github.com/cap-js/cds-dbs/issues/533)) ([e95fd17](https://github.com/cap-js/cds-dbs/commit/e95fd176094ac8b1d95b3bc68538057bf28aadf1))
* issue with reused select cqns ([#505](https://github.com/cap-js/cds-dbs/issues/505)) ([916d175](https://github.com/cap-js/cds-dbs/commit/916d1756422f0caf02c323052f2addafed39182a))
* joins without columns are rejected ([#535](https://github.com/cap-js/cds-dbs/issues/535)) ([eb9beda](https://github.com/cap-js/cds-dbs/commit/eb9beda728de60081d7afbfcd49305eeb241f3fb))
* mass insert for unknown entities ([#540](https://github.com/cap-js/cds-dbs/issues/540)) ([f2ea4af](https://github.com/cap-js/cds-dbs/commit/f2ea4affe65e76fa269766d3d02337ceb1138c65))
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: db-service/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.6.5](https://github.com/cap-js/cds-dbs/compare/db-service-v1.6.4...db-service-v1.6.5) (2024-03-05)
+
+
+### Fixed
+
+* deep delete for views without navigation ([#434](https://github.com/cap-js/cds-dbs/issues/434)) ([3ebc9c2](https://github.com/cap-js/cds-dbs/commit/3ebc9c29362045916c9060528729320c0eae4391))
```
Comment (patricebender) [inline db-service/CHANGELOG.md:12]: remove before release

File: db-service/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.6.5](https://github.com/cap-js/cds-dbs/compare/db-service-v1.6.4...db-service-v1.6.5) (2024-03-06)
+
+
+### Fixed
+
+* deep delete for views without navigation ([#434](https://github.com/cap-js/cds-dbs/issues/434)) ([3ebc9c2](https://github.com/cap-js/cds-dbs/commit/3ebc9c29362045916c9060528729320c0eae4391))
```
Comment (patricebender) [inline db-service/CHANGELOG.md:12]: remove before release ([release-please does not do this for us](https://github.com/googleapis/release-please/issues/296))

**Review by patricebender — CHANGES_REQUESTED** [review]

File: hana/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,13 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.0.7](https://github.com/cap-js/cds-dbs/compare/hana-v0.0.6...hana-v0.0.7) (2024-03-12)
+
```
Comment (patricebender) [inline hana/CHANGELOG.md:8]: we need to add https://github.com/cap-js/cds-dbs/pull/488 → release please seems to be buggy if a conventional commit contains code fences as described here: https://github.com/googleapis/release-please/issues/2020

**Comment by johannes-vogel** [comment]
I think we also need to increase peer dependency versions with this release.

**Review by patricebender — APPROVED** [review]

**Comment by johannes-vogel** [comment]
@patricebender Do we also need to update the PR description for the release tags?

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/db-service-v1.7.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/sqlite-v1.6.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/postgres-v1.6.0 :sunflower:

**Comment by cap-bots** [comment]
:robot: Release is at https://github.com/cap-js/cds-dbs/releases/tag/hana-v0.1.0 :sunflower:

**Pull request merged**